### ISSUE TITLE
old messages not shown

### ISSUE THREAD

**Issue by mirabilos** [k9mail/k-9#920]
I have set “Sync messages from” to “any time (no limit)” but still don’t get the one old message I can access via pine (a desktop IMAP client) to show up in K-9 Mail 5.006 (using the very same IMAP settings)…

The message in question is not even a month old!

**Comment by cketti** [comment]
Can you please create an empty folder, copy the message there, refresh the folder list in K-9 Mail, sync that new folder and check if K-9 Mail displays it there?

**Comment by mirabilos** [comment]
This is the INBOX, not some random folder… I need K-9 Mail to only synchronise the INBOX anyway…

**Comment by s3b-** [comment]
What cketti is probably trying to determine is whether the bug has something to do with the message itself or the folder it is kept in. Or something like that. Could you please try the steps cketti suggested and tell us the outcome?

Also, is this one old message the oldest message in the folder? Or is it just some random message k-9 decides not to show and there are several other older messages?

**Comment by mirabilos** [comment]
Ah well, I see.

The message in question was the *only* message in the INBOX.

I have created a folder via IMAP, moved the message there, spent about 5 minutes trying to figure out how to get K-9 Mail to show that folder, and, yes, it shows up there.

Then, I moved the message back to INBOX via pine, and it now shows up in INBOX in K-9 Mail again. Huh.

I admit this is extremely difficult to debug now…
